articles/iot-hub/quickstart-device-streams-echo-csharp.md

```diff
@@ -104,19 +104,21 @@ dotnet build
 
 # Run the application
 # In Linux/MacOS
-dotnet run $ServiceConnectionString MyDevice
+dotnet run "ServiceConnectionString" "MyDevice"
 
 # In Windows
-dotnet run %ServiceConnectionString% MyDevice
+dotnet run ServiceConnectionString MyDevice
```

Comment:
I meant these to be a %variable% in windows and $variable in Linux. But it might not have been clear.

You are right about the quotes though, if these are plain text and not a variable. I would suggest this change:

```
# Run the application
# In Linux/MacOS
dotnet run "<ServiceConnectionString>" "<deviceID>"

# In Windows
dotnet run <ServiceConnectionString> <deviceID>
```

Reply:
I agree